data-migration/2.0-to-3.0/harvest_project_info_nlesc.py

```diff
@@ -159,13 +162,63 @@ def get_project_related_projects(soup):
     return project_related_projects
 
 
-def get_project_image_url(soup):
-    for figure in soup.findAll("figure"):
-        for img in figure.findAll("img"):
-            imageUrl = img.get("src")
-            if imageUrl != "":
-                return make_url_secure(imageUrl)
-    return "https://FIXME"
+def get_project_image(soup, maxwidth=1024, maxheight=600):
+    def crop_to_aspect_ratio():
+        target_ar = maxwidth / maxheight
```

Comment:
change `target_ar` to `target_aspect_ratio`.

Reply:
Agree